sdk/core/azure-core/CHANGELOG.md

```diff
@@ -1,7 +1,10 @@
 # Release History
 
-## 1.13.1 (Unreleased)
+## 1.14.0 (Unreleased)
 
+### New Features
+
+- Added `azure.core.credentials.AzureNamedKeyCredential` credential #17548.
```

Comment:
emmm, wondering why azure-core changes are showing up here, I see your PR https://github.com/Azure/azure-sdk-for-python/pull/17548 already got merged

do we need to merge master to your dev branch first?